main.css

```diff
@@ -198,161 +235,193 @@ body {
 }
 
 .drive-with__woman {
-    width: 900px;
-    height: 940px;
-    padding-left: 15px;
+    width: 100%;
+    height: 100%;
 }
 
 .drive-with__feedback {
     display: flex;
     flex-direction: column;
     flex-wrap: wrap;
-    gap: 40px;
+    gap: 10px;
     justify-content: center;
     align-items: center;
+    text-align: center;
+    background-color: #F4F4F9;
 }
 
-.drive-with-feedback__text {
-    font-size: 50px;
-    font-weight: 400;
+.drive-with__feedback-text {
+    font-size: 20px;
+    font-weight: 40;
     line-height: 1.5;
-    width: 100%;
+    width: 70%;
+    margin-bottom: 50px;
 }
 
-.drive-with-feedback__italic {
-    font-weight: 300;
-    font-style: italic;
-    font-size: 40px;
+.drive-with__feedback-sign {
+    font-size: 20px;
+    font-weight: 40;
+    line-height: 1.5;
+    width: 100%;
+    margin: 0px;
 }
 
-.second__button-feedback-text {
-    font-size: 40px;
+.drive-with__feedback-text--italic {
+    font-weight: 30;
```

Comment:
30? there is no 30 in the [font-weight values](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#syntax)

Reply:
+. Changed to 300